src/ccache/config.cpp

```diff
@@ -522,6 +537,21 @@ home_directory()
 #endif
 }
 
+std::string
+atfile_format_to_string(AtFileFormat atfile_format)
+{
+  switch (atfile_format) {
+  case AtFileFormat::guess_from_compiler:
+    return "guess_from_compiler";
```

Comment:
This means that e.g. `ccache -p` can print `atfile_format = guess_from_compiler`, but that's not a valid setting according to `parse_atfile_format`. I suggest either using the empty string as the default or (probably better) making the default value token valid as a setting, similar to how `compiler_type` works. I would then also suggest `auto` instead of `guess_from_compiler` for symmetry with `compiler_type`.